api/doc/openapi.yaml

```diff
@@ -671,31 +641,27 @@ components:
           - vector: `vector<{non-reference MoveTypeId}>`
           - struct: `{address}::{module_name}::{struct_name}::<{generic types}>`
 
-
         Vector type value examples:
           * `vector<u8>`
           * `vector<vector<u64>>`
           * `vector<0x1::DiemAccount::Balance<0x1::XDX::XDX>>`
 
-
         Struct type value examples:
           * `0x1::Diem::Diem<0x1::XDX::XDX>`
           * `0x1::Abc::Abc<vector<u8>, vector<u64>>`
           * `0x1::DiemAccount::AccountOperationsCapability`
 
-
         Note:
           1. Empty chars should be ignored when comparing 2 struct tag ids.
           2. When used in an URL path, should be encoded by url-encoding (AKA percent-encoding).
-      example: >-
-        0x1::XUS::XUS
+      example: "0x1::XUS::XUS"
```

Comment:
switch to one line, so that when the example is included in another struct, the text won't include "\n"